climpred/classes.py

```diff
@@ -84,6 +87,43 @@ def _display_metadata(self):
     return summary
 
 
+def _display_metadata_html(self):
+    header = f'<h4>climpred.{type(self).__name__}</h4>'
+    display_html(header, raw=True)
+    init_repr_str = dataset_repr(self._datasets['initialized'])
+    init_repr_str = init_repr_str.replace(
+        'xarray.Dataset', 'climpred Initialized : xarray.Dataset'
```

Comment:
For consistency with the text repr, I would change `climpred Initialized  : xarray.Dataset` to just `Initialized Ensemble`.